src/app.js

```diff
@@ -130,11 +130,26 @@ class SAFEApp extends EventEmitter {
   * Helper to lookup a given `safe://`-url in accordance with the
   * convention and find the requested object.
   *
+
   * @param {String} url the url you want to fetch
+  * @param {Object} options options Object
+  *                       range: { object } override header range with bytes from request.
```

Comment:
I would define a custom object for documenting the options, let's say `WebFetchOptions`.